service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -185,9 +185,11 @@ private void publishPlacementApplicationUploadEvent(CallbackRequest callbackRequ
             new PlacementApplicationEvent(callbackRequest, requestData.authorisation(), requestData.userId()));
     }
 
-    //TODO: refactor logic. Double negative for !isNotEmpty currently. FPLA-1473
-    private boolean isUpdatingExistingPlacement(Placement previousPlacement, Placement newPlacement) {
-        return isNotEmpty(previousPlacement)
-            && newPlacement.getApplication().equals(previousPlacement.getApplication());
+    private boolean isNewPlacement(Placement previousPlacement) {
+        return isEmpty(previousPlacement);
+    }
```

Comment:
you think this needs to be extracted or we can just do isEmpty(previousPlacement) || isUpdated(...) in above if

Reply:
i thought it will be clear to see it as new or updated. can move it to if condition

Reply:
was an open Q, but sure :)